keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -107,28 +108,68 @@ As part of this proposal, we are mainly proposing three changes:
    - NodeExpansionFailed // state set when expansion has failed in kubelet with a terminal error. Transient errors don't set NodeExpansionFailed.
 3. Update quota code to use `max(pvc.Spec.Resources, pvc.Status.AllocatedResources)` when evaluating usage for PVC.
 
+### Making resizeStatus more general
+
+After some discussion with sig-storage folks and to accommodate changes coming from https://github.com/kubernetes/enhancements/issues/3751 we are proposing that we rename `pvc.Status.ResizeStatus` to `pvc.Status.AllocatedResourceStatus` and make it a map.
+
+So basically new API will look like:
+
+```go
+type ClaimResourceResizeStatus string
```

Comment:
Do you think we would ever want the status to be more than a string? ie, something more condition-like with a timestamp?

Reply:
That could be useful for debugging purposes, but not useful for driving the state machine.  Since this field was added specifically to drive the statemachine, I think we will not necessarily need timestamps here.

Reply:
Should we leave room to add it in the future, or include it for general observability? We are looking at adding timestamps in other areas already: https://github.com/kubernetes/enhancements/pull/3796

Reply:
Hmm, but pvc already has `pvc.Status.Conditions` - so if we need something there for informational purposes we could extend the conditions. PV object didn't had conditions, so there is that.